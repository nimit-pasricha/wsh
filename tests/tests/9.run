../src/mysh tests/9.sh