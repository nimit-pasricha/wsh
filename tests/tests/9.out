e: aliased to 'echo'
alias: shell builtin
which: shell builtin
path: shell builtin
cd: shell builtin
history: shell builtin
exit: shell builtin
unalias: shell builtin
echo: found at /bin/echo
cat: found at /bin/cat
/bin/echo: found at /bin/echo
./notarealfile: not found
notacommand: not found
e: aliased to 'exit'
echo: not found
cat: not found
echo: found at /bin/echo
